evalai/utils/challenges.py

```diff
@@ -194,6 +194,48 @@ def display_participated_or_hosted_challenges(is_host=False, is_participant=Fals
             echo("Sorry, no challenges found!")
 
 
+def pretty_print_challenge_details(challenge):
+    table = BeautifulTable(max_width=210)
+    attributes = ["description", "submission_guidelines", "evaluation_details", "terms_and_conditions"]
+    column_attributes = ["Title", "Start Date", "End Date", "Description", "Submission Guidelines",
+                         "Evaluation Details", "Terms and Conditions"]
+    table.column_headers = column_attributes
```

Comment:
Why not directly set `table.column_headers` to the list? I don't see `column_attributes` being used elsewhere.

Reply:
Didn't notice that. Will change it!

Reply:
Done